nixos/modules/services/security/vaultwarden/default.nix

```diff
@@ -86,6 +86,14 @@ in {
       '';
     };
 
+    dataFolder = mkOption {
+      type = str;
+      default = "/var/lib/bitwarden_rs";
+      description = ''
+        The directory under which vaultwarden save it's data.
+      '';
```

Comment:
```suggestion
      description = ''
        The directory under which vaultwarden save its data.

        <note><para>
          If left as the default value this directory will automatically be created before
          the vaultwarden server starts, otherwise the sysadmin is responsible
          for ensuring the directory exists with appropriate ownership and permissions.
        </para></note>
      '';
```